_includes/masthead.html

```diff
@@ -4,7 +4,36 @@
     <div class="masthead-copy container-fluid">
         <h2 class="masthead-title">Welcome to {{ site.title }}</h2>
 
-        <h4 class="masthead-description">NewHaven.io is a nonprofit 501(c)(6) developer group in New Haven, CT. If you’re interested in programming, the intertubes, and whatnot, this is the group for you.</h4>
+        <h4 class="masthead-description">  newhaven.io is...
+            <ul>
+            <li><a href="http://newhaven.io/blog/what-is-NewHavenIO/" rel="nofollow">connecting and supporting</a> New Haven’s technology community</li>
+            <li>self-managed by volunteers (and always looking for more!)</li>
+            <li>a 501(c)(6) non-profit</li>
+            <li>about tech. all tech. your tech.</li>
+            </ul>
+            <i>How can we help you?</i>
+            <hr>
+            <p>If you’re a <strong>developer</strong>:</p>
+            <ul>
+            <li><a href="https://www.meetup.com/newhavenio/" rel="nofollow">Community</a> (socials)</li>
+            <li>Grow: A chance to learn, to mentor, to present</li>
+            <li><a href="https://newhavenio-slackin.herokuapp.com/" rel="nofollow">Network</a>: find a new team, a new teammate, or a co-founder</li>
+            </ul>
+            <p>If you’re a local software <strong>meetup group</strong>:</p>
+            <ul>
+            <li>Wider outreach, visibility</li>
+            <li>Access to meeting spaces</li>
+            <li>Logistical help</li>
+            </ul>
+            <p>If you’re a software <strong>company</strong>:</p>
+            <ul>
+            <li>Recruiting, hiring</li>
+            <li>A way for your employees to grow and learn</li>
+            </ul>
+            <p>If you’re an <strong>entrepreneur</strong>:</p>
+            <ul>
+            <li>Sponsor a meetup (<em>usually between $300-$500</em>), and we’ll let you pitch to the developers.</li>
```

Comment:
Is sponsorship something we should consider here? Cc @ZachBeta

Reply:
I think of it like a FAQ. IMO @danbernier was wise to include it.  I suspect it'll obviate the need to repeatedly answer a lot of recurring questions, including that particular one.

Reply:
🤔 I wasn't aware that we had settled on the sponsorship for pitching the crowd direction. Maybe I missed that convo. Either way, if that's settled, then happy to include it